examples/dynamic_behavior/skip_messages.cpp

```diff
@@ -31,12 +31,12 @@ void caf_main(actor_system& system) {
   scoped_actor self{system};
   self->request(serv, std::chrono::seconds(10), ping_atom_v)
     .receive(
-      [&](pong_atom) {
+      [&self, worker](pong_atom) {
```

Comment:
```suggestion
      [self, worker](pong_atom) {
```

Reply:
This is a `scoped_actor`. You actually *have* to capture this by reference, since you cannot copy it.